File: backend/print_tenants.py
import json
import codecs
try:
    with open('tenants_dump.json', 'r', encoding='utf-16') as f:
        data = json.load(f)
        for t in data:
            print(f"Name: {t.get('displayName')}")
            print(f"  Tenant: {t.get('tenantId')}")
            print(f"  Client: {t.get('clientId')}")
            print("-" * 20)
except Exception as e:
    print(f"Error: {e}")

File: backend/print-tenants.js
const fs = require('fs');
try {
    const raw = fs.readFileSync('tenants_dump.json', 'utf16le');
    const data = JSON.parse(raw);
    data.forEach(t => {
        console.log(`Name: ${t.displayName}`);
        console.log(`  Tenant: ${t.tenantId}`);
        console.log(`  Client: ${t.clientId}`);
        console.log('-------------------');
    });
} catch (e) {
    console.error('Failed to parse:', e.message);
}
